nni/retiarii/experiment.py

```diff
@@ -125,22 +125,22 @@ def _start_strategy(self):
         except Exception as e:
             _logger.error('Your base model cannot be parsed by torch.jit.script, please fix the following error:')
             raise e
-        base_model = convert_to_graph(script_module, self.base_model, self.recorded_module_args)
+        base_model_ir = convert_to_graph(script_module, self.base_model)
 
-        assert id(self.trainer) in self.recorded_module_args
-        trainer_config = self.recorded_module_args[id(self.trainer)]
-        base_model.apply_trainer(trainer_config['modulename'], trainer_config['args'])
+        recorded_module_args = get_records()
+        trainer_config = recorded_module_args[id(self.trainer)]
```

Comment:
better to add error message here to tell users that they may forgot to decorate their customized trainer